docs/UserGuide.md

```diff
@@ -47,7 +47,9 @@ Example:
  
 ### Adding a task: `add`
 Adds a task into the task manager.<br>
-Format: `add NAME [from/at START_DATE START_TIME][to/by END_DATE END_TIME][repeat every RECURRING_INTERVAL][-PRIORITY]`
+Format: `add NAME, [from/at START_DATE START_TIME][to/by END_DATE END_TIME][repeat every RECURRING_INTERVAL][-PRIORITY]`
+
+>We do not require an explicit command for `add`. We make it the default thing to do, when you type in anything! Hence typing in `add` itself is optional. However, if you want to add a task that starts with other command words, please include the `add` to override the other command words!
 
```

Comment:
Let the user know of the default command. I have also removed the commands below to exclude `add`. Maybe can add in examples to show when this is necessary for user.